Libraries/LibWebView/Application.cpp

```diff
@@ -172,6 +173,7 @@ ErrorOr<void> Application::initialize(Main::Arguments const& arguments)
     args_parser.add_option(dns_server_port, "Set the DNS server port", "dns-port", 0, "port (default: 53 or 853 if --dot)");
     args_parser.add_option(use_dns_over_tls, "Use DNS over TLS", "dot");
     args_parser.add_option(validate_dnssec_locally, "Validate DNSSEC locally", "dnssec");
+    args_parser.add_option(webserial_device_path, "Enables the WebSerial API and sets device to use", "webserial-device-path", 0, "path");
```

Comment:
I imagine that it's not the goal to use this command line argument, but rather let serial_cpp enumerate available ports and let the user select/confirm the one to use. Could you add a FIXME somewhere to that effect?

Reply:
Indeed, a device picker like Chrome has would be really neat to have. I added a NOTE for this at https://github.com/LadybirdBrowser/ladybird/pull/5807/files#diff-2bea0dcc11008cdc88cc922f08d9383be7131891361a869cf186bd3a8782a54cR101, would you like me to reword that a bit and change it into a FIXME?